_docker/tailscale-entrypoint.sh

```diff
@@ -55,7 +55,8 @@ TAILSCALED_PID="${!}"
 tailscale up \
   --hostname "${TAILSCALE_HOSTNAME}" \
   --authkey "${TAILSCALE_AUTH_KEY}" \
-  --advertise-routes "${TAILSCALE_ADVERTISE_ROUTES}"
+  --advertise-routes "${TAILSCALE_ADVERTISE_ROUTES}" \
+  ${TAILSCALE_ADDITIONAL_FLAGS}
```

Comment:
Just confirming, this is intentionally not quoted

Reply:
I think it has to be unquoted for things to work as expected, or you end up with this kind of thing:
```
$ sudo tailscale up "--advertise-routes=192.168.42.0/24 --advertise-exit-node"
"192.168.42.0/24 --advertise-exit-node" is not a valid IP address or CIDR prefix
```

Reply:
Totally, it's "preferable" to always quote variables in shell scripts but this one is an exception where quoting (as you point out is wrong).

Preferable because it allows stricter control over inputs.